woocommerce/woocommerce-ios
# Lightweight Storefront: Theme Picker UI

Closes: #11326


## Description


This adds the SwiftUI view for the theme carousel, as well as the SwiftUI view for the Profiler theme picker (which makes use of the aforementioned carousel).

It uses example URLs to show the screenshots and should be replaced with real data.

- ThemeCarouselView` is meant to be a reusable View as there's two places that will use it, Profiler and Settings.
- `ProfilerThemesPickerView` is meant to show how to make use of `ThemeCarouselView`


## Testing instructions


They're not used in the app yet, so test by previewing `ThemesCarouselView` and `ProfilerThemesPickerView`. Especially please test the scrolling behavior and make sure the last message is also shown.


| Carousel default view | Carousel missing image | Carousel last message |
|--------|--------|--------|
| <img width="342" alt="Screenshot 2023-12-01 at 13 15 59" src="https://github.com/woocommerce/woocommerce-ios/assets/266376/e3d789bc-36a6-44ed-bd0e-a97b835a7e7e"> | <img width="337" alt="Screenshot 2023-12-01 at 13 16 12" src="https://github.com/woocommerce/woocommerce-ios/assets/266376/4f36567d-4dee-42e1-a8bf-3705e4f56ed8"> | <img width="335" alt="Screenshot 2023-12-01 at 13 16 18" src="https://github.com/woocommerce/woocommerce-ios/assets/266376/239e8285-a409-4361-91f8-412f65ddb9a7">  |

`ProfilerThemesPickerView`:

<img width="365" alt="Screenshot 2023-12-01 at 13 16 33" src="https://github.com/woocommerce/woocommerce-ios/assets/266376/8221e10f-3d14-4b6d-b213-95b1994262ea">
